zulip_bots/zulip_bots/lib.py

```diff
@@ -198,6 +251,18 @@ def handle_message(message):
                 return
 
         if is_private_message or is_mentioned:
+            # Handle any default_commands first
+            if len(default_commands) > 0:
+                if '' in default_commands and len(message['content']) == 0:
+                    restricted_client.send_reply(message, default_commands[''][0]())
+                    return
+                for command in default_commands:
+                    if command == '':
+                        continue
+                    if message['content'].startswith(command):
```

Comment:
Yeah, I think we want an equality check, not startswith.  It also means `''` doesn't need to be a special case.